### PR TITLE
Add Json output option to showing a topic

### PR DESCRIPTION
**Pull request by cody-code-wy**
The json object returned has the keys
`real`: _Content of the real tweet_
`fake`: _Content of the fake tweet_
`real_handle`: _Handle of user who created the real tweet_
`fake_handle`: _Handle of user who created the fake tweet_
`tweet_topic`: _Topic of the tweets sent_
`topic`: topic the user had selected_

### CODE REVIEW

File: app/actions.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+require 'json'
```
Comment (tjukes) [inline app/actions.rb]: Shouldn't this be required in one of the global config files? We'll probably need it in the views as well.

File: app/actions.rb [hunk 2]
```diff
@@ -11,7 +12,18 @@
   end
   @real = @topics.sample.twitter_handles.where("real_twitter_handle_id is null").sample.tweets.sample
   @fake = TwitterHandle.where("real_twitter_handle_id = ?", [@real.twitter_handle.id]).sample.tweets.sample
-  erb :'topics/show.html'
+  unless params["json"]
```
Comment (tjukes) [inline app/actions.rb]: Having an unless/else statement is not conventional.

File: app/actions.rb (hunk 1 above)
Comment (cody-code-wy) [inline app/actions.rb]: Turns out it already required. This is not nessisary

File: app/actions.rb (hunk 2 above)
Comment (cody-code-wy) [inline app/actions.rb]: @tjukes actually `unless` with an `else` is convention when the code in else is the secondary path.

File: app/actions.rb [hunk 3]
```diff
@@ -11,7 +12,18 @@
...
+  else
+    {
+      real: @real.content,
+      fake: @fake.content,
+      topic: id,
+      real_handle: @real.twitter_handle.twitter_handle,
+      fake_handle: @fake.twitter_handle.twitter_handle,
+      tweet_topic: @real.twitter_handle.topic.topic,
+    }.to_json
+  end
```
Comment (tjukes) [inline app/actions.rb]: Not clear to me what this block is doing. Could you add more context in the commit message?

File: app/actions.rb (hunk 1 above)
Comment (tjukes) [inline app/actions.rb]: Even better!

File: app/actions.rb (hunk 2 above)
Comment (tjukes) [inline app/actions.rb]: @cody-code-wy shouldn't it still be an if/else? unless/else reads much less clearly.

File: app/actions.rb (hunk 3 above)
Comment (cody-code-wy) [inline app/actions.rb]: @tjukes its not a block, it's a hash

Comment (cody-code-wy) [inline app/actions.rb]: @tjukes it is the content of the json response.

File: app/actions.rb (hunk 2 above)
Comment (cody-code-wy) [inline app/actions.rb]: @tjukes while I agree that it reads less clearly, it is better to use unless here so the erb line is the first thing a person reading the code sees.

**Pull request closed without merging**